### PR TITLE
Fix 404 link for android setup

### PR DESCRIPTION
**Pull request by FeiChen-okta**
### Resolves:

* [OKTA-330023](https://oktainc.atlassian.net/browse/OKTA-330023)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/configure-packages/android/installsdk.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Use the [Okta OIDC](https://github.com/okta/okta-oidc-android) library, available through [JCenter](https://bintray.com/okta/com.okta.oidc.android/okta-oidc-android).
+Use the [Okta OIDC](https://github.com/okta/okta-oidc-android) library, available through [JCenter](https://bintray.com/okta/com.okta.android/okta-oidc-android).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/configure-packages/android/installsdk.md:1]: don't need the comma

Use the Okta OIDC library available through JCenter.

**Review by susanharper-okta — COMMENTED** [review]
1 minor change.

**Comment by FeiChen-okta** [comment]
> 1 minor change.

Fixed. I also updated the version to the latest sdk release

**Review by susanharper-okta — APPROVED** [review]
LGTM 🚢

**Pull request merged**